language-examples/python/vectara-grpc/grpc_basic_operations.py

```diff
@@ -172,14 +176,27 @@ def query(customer_id: int, corpus_id: int, query_address: str, jwt_token: str,
         response = query_stub.Query(batch_request,
                                     credentials=grpc.access_token_call_credentials(jwt_token),
                                     metadata=[("customer-id-bin", packed_customer_id)])
+        if (response.status and
+            any(status.code != status_pb2.StatusCode.OK
+                for status in response.status)):
+            logging.error("Query failed with response: %s", response.status)
+            return response.status, False
+
+        for response_set in response.response_set:
+            for status in response_set.status:
+                if status.code != status_pb2.StatusCode.OK:
```

Comment:
I think this is okay for now, at least it shows the correct pattern.
There are certain response codes like ALREADY_EXISTS, which wouldn't be considered errors in a lot of scenarios (but might in some)

Reply:
Added some ALREADY_EXISTS checks in places.